src/utils/externalLinks.js

```diff
@@ -1,21 +1,16 @@
-import i18next from 'i18next';
 import history from '../history';
 import routesReg from './routes';
-import { errorToastDisplayed } from '../actions/toaster';
-import store from '../store';
 
 export default {
   init: () => {
     const { ipc } = window;
 
     if (ipc) {
       ipc.on('openUrl', (action, url) => {
-        const normalizedUrl = url.toLowerCase().replace('lisk://', '/');
+        const normalizedUrl = url.replace('lisk://', '/');
         const route = routesReg.find(item => item.regex.test(normalizedUrl));
         if (route !== undefined) {
           history.push(normalizedUrl);
-        } else {
-          store.dispatch(errorToastDisplayed({ label: i18next.t('The URL was invalid') }));
```

Comment:
Please be aware that removing this line, we won't be notifying the user about probable mistakes in URL. for example, `lisk://mai/transactions` or `lisk://main/transaction` will result in no action and can be confusing.